cmake/MacOSX/fix_dependencies.rb

```
@@ -13,511 +13,504 @@
-    newFrameworkPath="#{targetPath}/#{libname}"
-  end
 
-  # update ids of dependencies
-  $currentIndent-=1
+    # update ids of dependencies
+    @currentIndent-=1
 
-  return newFrameWorkPath, libname
-end
+    return newFrameWorkPath, libname
+  end
```

RuboCop (1.82.1)

[convention] Assignment Branch Condition size for `handleFramework` is too high. [<18, 38, 11> 43.46/23]

(Metrics/AbcSize)